homu/server.py

```diff
@@ -1,7 +1,8 @@
 import hmac
 import json
 import urllib.parse
-from .main import PullReqState, parse_commands, db_query, INTERRUPTED_BY_HOMU_RE, synchronize
+from .main import (PullReqState, parse_commands, db_query,
+                   INTERRUPTED_BY_HOMU_RE, synchronize)
```

Comment:
Do you think this is worth going back through and tidying right now, or shall we land these fixups as-is and catch the diff noise reduction cleanups in future refactors?